cl/simple_pages/views.py

```diff
@@ -344,7 +351,7 @@ async def contribute(request: HttpRequest) -> HttpResponse:
 
 
 @ratelimiter_unsafe_3_per_m
-def contact(
+async def contact(
```

Comment:
I believe this view will raise the ValueError described in https://github.com/freelawproject/courtlistener/issues/2930 because of the ratelimit decorator

Reply:
I think it's better to leave it because it still basically works. The difference is it protects the site by crashing instead of working, but that's still something. Of course, it is better to work properly!